tests/helpers/stubs.py

```diff
@@ -131,11 +131,12 @@ class FakeUrl:
 
     """QUrl stub which provides .path(), isValid() and host()."""
 
-    def __init__(self, path=None, valid=True, host=None, url=None):
+    def __init__(self, path=None, valid=True, host=None, url=None, scheme = None):
```

Comment:
There typically are no spaces around `=` for default arguments in Python, so that should be `scheme=None`.

Reply:
Will do!